### PR TITLE
(DOCS-3818) Daily usage region availability

### PR DESCRIPTION
**Pull request by hestonhoffman**
See [DOCS-3818](https://datadoghq.atlassian.net/browse/DOCS-3818).

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/billing/usage_attribution.md
Comment (kayayarai) [inline content/en/account_management/billing/usage_attribution.md:101]:
```suggestion
Daily usage attribution is only available for organizations in a US1 or EU1 datacenter. On US3, US5, and FedGov, you can instead download a report.
```

Comment (kayayarai) [reply content/en/account_management/billing/usage_attribution.md:101]: I added this because otherwise it seems vague as to whether these customers, who might not know exactly what "Daily usage attribution" looks like, can do the next steps (they can).

File: content/en/account_management/billing/usage_attribution.md
Comment (kayayarai) [inline content/en/account_management/billing/usage_attribution.md:87]: Is .tsv correct. Or should it be `.csv`? And if you go with the monospace file extension wording rather than the acronym, add "file" after: `downloaded as a .csv file` or `downloaded as a CSV.`

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/account_management/billing/usage_attribution.md
Comment (hestonhoffman) [reply content/en/account_management/billing/usage_attribution.md:87]: Oh wow, I completely missed this paragraph in the original doc. I'll download one and check

File: content/en/account_management/billing/usage_attribution.md
Comment (katherinebroner) [reply content/en/account_management/billing/usage_attribution.md]: @kayayarai what specifically you referring to for "download a report"? US3, US5 and FedGov customers cannot download the daily usage attribution reports because the feature is deprecated for them. However, they are still able to download the Monthly usage attribution data via CSV or access hourly usage attribution data via API

File: content/en/account_management/billing/usage_attribution.md
Comment (katherinebroner) [reply content/en/account_management/billing/usage_attribution.md:87]: @kayayarai `.tsv` is correct

**Comment by hestonhoffman** [comment]
Hi @kayayarai, 

@katherinebroner and I decided to remove the section for other regions instead of using a note. So this is good for another look. I added a little clarification on the TSV stuff after taking a look at the feature. This feature does not use CSVs.

**Review by kayayarai — APPROVED** [review]
Nice! Much slicker. I'll let Katherine approve too before I merge

**Review by katherinebroner — APPROVED** [review]
looks good, thanks!

**Pull request merged**
